harper-core/Cargo.toml

```diff
@@ -24,6 +24,7 @@ thiserror = "2.0.9"
 unicode-blocks = "0.1.9"
 unicode-width = "0.2.0"
 levenshtein_automata = { version = "0.2.1", features = ["fst_automaton"] }
+typst-syntax = "0.12.0"
```

Comment:
Typst is pretty big, and not applicable to some applications (i.e. Obsidian). Would it be possible to put Typst support behind a feature flag?